### DISCUSSION TITLE
[FE101] CSS selector 權重問題

### DISCUSSION THREAD

**Discussion by yichennnn36** [Lidemy/forum#117] [前端相關問題] [answered]
想問一下關於 `!important` 的使用：

以下使用 !important 是會成功顯示黑色

<img width="579" alt="1" src="https://user-images.githubusercontent.com/82022020/118474948-3b8ec600-b73e-11eb-8ae2-203d48f4a32a.png">

改成最外層的 .wrapper 是會被蓋掉的的原因是什麼呢？權重不是 (1, 0, 0, 0, 0) 嗎？

<img width="589" alt="2" src="https://user-images.githubusercontent.com/82022020/118475023-53664a00-b73e-11eb-8904-a7a074570db3.png">

感謝大家！

**Comment by cmtilo** [comment]
https://yachuhsu1992.medium.com/css-selectors-note-1-2d4f0e0c1233
這篇不知道有沒有幫助...提供參考

**Reply by yichennnn36** [reply]
感謝分享！

**Comment by aszx87410** [comment, marked as answer]
所謂的權重是用在同一個元素時才會有權重，你改掉之後就不是同個元素了

**Reply by yichennnn36** [reply]
原來是這樣！剛去檢查一下發現問題了
感謝老師～～
